source/manual/business-readiness-finder.html.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
I'm not convinced this page is needed.

I think if you move the sections about "Facets" and "Grouped display" to finder-frontend, and remove the section about the "Email signup page", what's left (the Overview) could be restored to its original home: [Publish changes to the business readiness finder](https://docs.publishing.service.gov.uk/manual/business-readiness-publish-changes.html)